reana_workflow_controller/rest.py

```diff
@@ -1226,11 +1224,15 @@ def set_workflow_status(workflow_id_or_name):  # noqa
             return jsonify(
                 {'message': 'User {} is not allowed to access workflow {}'
                  .format(user_uuid, workflow_id_or_name)}), 403
-        operational_parameters = {}
-        if request.json:
-            operational_parameters = request.json.get('operational_parameters')
         if status == START:
+            operational_parameters = {} if not request.json else \
+                request.json.get('operational_parameters', {})
             return _start_workflow(workflow, operational_parameters)
+        elif status == DELETED:
+            all_runs = True if request.json.get('all_runs') == 1 else False
```

Comment:
I would go for `true/false` see [OpenAPI docs](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types) and [JSON data types](https://www.w3schools.com/js/js_json_datatypes.asp)... It will ease coding the UI and it will be more understandable.

Reply:
Fixed